### PR TITLE
Add recipe for shippinglabel

### PR DESCRIPTION
**Pull request by domdfcoding**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

`shippinglabel` vendors [trove-classifiers](https://pypi.org/project/trove-classifiers/) because there are issues with building it from source. The license for `trove-classifiers` is in the `shippinglabel/_vendor/trove_classifiers` directory.

### CODE REVIEW

**Comment by domdfcoding** [comment]
@conda-forge-admin please ping team

File: recipes/shippinglabel/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    - shippinglabel
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/domdfcoding/shippinglabel
+  summary: Utilities for handling packages.
+  license: MIT
```
Comment (dopplershift) [inline recipes/shippinglabel/meta.yaml:42]: Looks like the trove license is apache 2.0:
```suggestion
  license: MIT AND Apache-2.0
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
Just a minor license tweak.

File: recipes/shippinglabel/meta.yaml (hunk 1 above)
Comment (domdfcoding) [reply recipes/shippinglabel/meta.yaml]: Thanks. Wasn't quite sure what the syntax was for that.

**Review by dopplershift — APPROVED** [review]

**Comment by domdfcoding** [comment]
Thanks for the help @dopplershift

**Pull request merged**